Logic/Path/vectors.py

```diff
@@ -0,0 +1,85 @@
+'''Vector Handling Module'''
+
+import sys
+import math
+import numpy as np
+
+
+class Vectors:
+    '''Responsible for handling vector calculation'''
+
+    @staticmethod
+    def distance_from_two_points(point_one, point_two):
+        '''Calculates the distances between two points'''
+
+        return math.sqrt(((point_one[0] - point_two[0]) ** 2) + ((point_one[1] - point_two[1]) ** 2))
+    
+    @staticmethod
+    def move_from_two_points(point_one, point_two, distance):
+        '''Responsible for moving a given distance between two points'''
```

Comment:
Maybe change to 'Responsible for moving along a vector given two points'?

Reply:
Done